src/javascript/app/ng-wig/ng-wig-plugin-adapter.component.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('component: ngWigPlugin', () => {
+    let component;
+    let scope;
+    let plugin;
+    let execCommand;
+    let editMode;
+    let disabled;
+    let options;
+    let content;
+    let element;
+    let compile;
+
+    beforeEach(module('ngWig'));
+
+    beforeEach(inject((_$componentController_, _$rootScope_, _$compile_) => {
+        scope = _$rootScope_.$new();
+        compile = _$compile_;
+
+        plugin = {
+            name: 'myPlugin',
+            pluginName: 'my-plugin'
+        };
+        editMode = false;
+        disabled = false;
+        options = ['Option 1', 'Option 2'];
+        content = 'Fake text';
+        element = angular.element('<div></div>');
+        execCommand = 'fakeCmd()';
+
+        spyOn(element, 'replaceWith');
+        
+        component = _$componentController_('ngWigPlugin', 
+                                            { $scope: scope, $element: element }, 
+                                            { plugin: plugin, execCommand: execCommand, editMode: editMode, disabled: disabled, options: options, content: content }
+                                            );
+    }));
+    
+    it('should expose plugin', () => {
+        expect(component.plugin).toEqual(plugin);
```

Comment:
I followed the guidelines from AngularJS official website about [components](https://docs.angularjs.org/guide/component), particularly the section that describes **Unit-testing Component Controllers**